tst/EnergyPlus/unit/DXCoils.unit.cc

```diff
@@ -279,3 +279,525 @@ namespace EnergyPlus {
 	}
 
 }
+
+TEST( DXCoilsTest, TestMultiSpeedDefrostCOP )
+{
+	// Test that the COP calculation is correct when the defrost is on. #4973
+	
+	using CurveManager::Quadratic;
+	using CurveManager::BiQuadratic;
+	using CurveManager::NumCurves;
+	using EnergyPlus::DataEnvironment::OutHumRat;
+	using EnergyPlus::DataEnvironment::OutDryBulbTemp;
+	using EnergyPlus::DataEnvironment::OutBaroPress;
+	using DXCoils::CalcMultiSpeedDXCoilHeating;
+	int DXCoilNum;
+	
+	// Set up heating coil and curves.
+	
+	InitializePsychRoutines();
+	NumDXCoils = 1;
+	DXCoilNum = 1;
+	DXCoil.allocate( NumDXCoils );
+	DXCoilData & Coil = DXCoil( DXCoilNum );
+	
+	Coil.DXCoilType = "Coil:Heating:DX:MultiSpeed";
```

Comment:
@nmerket Also, you could change all this follow code to build out the data structure of the Coil:Heating:DX:MultiSpeed and curves to use the new process_idf() unit test helper function. Examples are [here](https://github.com/NREL/EnergyPlus/blob/41b2d221b08550306859c207782e07ae05e18b93/tst/EnergyPlus/unit/Humidifiers.unit.cc#L151-186), [here](https://github.com/NREL/EnergyPlus/blob/41b2d221b08550306859c207782e07ae05e18b93/tst/EnergyPlus/unit/Humidifiers.unit.cc#L188-246), or [here](https://github.com/NREL/EnergyPlus/blob/41b2d221b08550306859c207782e07ae05e18b93/tst/EnergyPlus/unit/MixedAir.unit.cc#L15-98). That said, since you already have all this code completed and functioning it is not necessary to change to the new process_idf() approach. I would still change to the HVACFixture though to fix the compile errors and make sure all clean up of state variables happens automatically.